tls/s2n_signature_scheme.c

```diff
@@ -308,15 +270,10 @@ const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_20230317[] = {
     &s2n_rsa_pkcs1_sha512,
 
     /* TLS1.2 with ECDSA */
-    &s2n_ecdsa_sha256, /* same iana value as TLS 1.3 s2n_ecdsa_secp256r1_sha256 */
-    &s2n_ecdsa_sha384, /* same iana value as TLS 1.3 s2n_ecdsa_secp384r1_sha384 */
+    &s2n_ecdsa_sha256,
+    &s2n_ecdsa_sha384,
     &s2n_ecdsa_sha512,
 
-    /* TLS1.3 with ECDSA */
-    &s2n_ecdsa_secp256r1_sha256,
-    &s2n_ecdsa_secp384r1_sha384,
-    &s2n_ecdsa_secp521r1_sha512,
-
```

Comment:
Is this strict TLS 1.2 then as the comment suggests? Previously it was both TLS 1.3 and 1.2 right.

Should the comments be updated to just say `ECDSA`

Reply:
Oh nice catch, I missed all those comments